bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/BoschSHCDeviceHandler.java

```diff
@@ -83,9 +87,44 @@ public void initialize() {
      *         otherwise
      */
     protected boolean processDeviceInfo(Device deviceInfo) {
+        try {
+            updateLocationPropertiesIfApplicable(deviceInfo);
+        } catch (Exception e) {
```

Comment:
I think this would be sufficient?
```suggestion
        } catch (BoschSHCException e) {
```

otherwise you could hide bugs in case a `RuntimeException` is thrown.